js/script.js

```diff
@@ -34,224 +33,94 @@ let messDict =[
+    1: { time: 13.5, meal: "Lunch", text: "Working day 11:30 - 1:30, Holiday 12:00 - 1:30" },
+    2: { time: 17.5, meal: "Snacks", text: "4:30 - 5:30" },
+    3: { time: 21, meal: "Dinner", text: "7:30 - 9:00" },
 }
 
-const setLunchRoutine = () => {
-    let header = document.createElement("H3")
-    let headerText = document.createTextNode("LUNCH (Working day 11:30 - 1:30, Holiday 12:00 - 1:30)")
-    header.appendChild(headerText)
-    document.getElementById("header").appendChild(header)
-    
-    messDict[day][1].forEach(e => {
-        let node = document.createElement("li")
-        let textnode = document.createTextNode("🔷 "+ e)
-        node.appendChild(textnode)
-        node.setAttribute("class", "list-group-item");
-        document.getElementById("foodlist").appendChild(node);
-    })
-}
-
-const setSnacksRoutine = () => {
-    let header = document.createElement("H3")
-    let headerText = document.createTextNode("Snacks (4:30 - 5:30)")
-    header.appendChild(headerText)
-    document.getElementById("header").appendChild(header)
+let now = new Date();
+let day = now.getDay();
+let hour = now.getHours();
+let minute = now.getMinutes();
 
-    messDict[day][2].forEach(e => {
-        let node = document.createElement("li")
-        let textnode = document.createTextNode("🔷 "+ e)
-        node.appendChild(textnode)
-        node.setAttribute("class", "list-group-item");
-        document.getElementById("foodlist").appendChild(node);
-    })
-}
+// 0: breakfast, 1: lunch, 2: snacks, 3: dinner
+let meal = 0;
 
-const setDinnerRoutine = () =>{
-    let header = document.createElement("H3")
-    let headerText = document.createTextNode("DINNER 7:30 - 9:00")
-    header.appendChild(headerText)
-    document.getElementById("header").appendChild(header)
-    
-    messDict[day][3].forEach(e => {
-        let node = document.createElement("li")
-        let textnode = document.createTextNode("🔷 "+ e)
-        node.appendChild(textnode)
-        node.setAttribute("class", "list-group-item");
-        document.getElementById("foodlist").appendChild(node);
-    })
+const returnWeekDay = (day) => {
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    return days[day];
 }
 
-const returnWeekDay = (dayNumber) => {
-    switch (dayNumber) {
-        case 0:
-            return "Sunday"
-        case 1:
-            return "Monday"
-        case 2:
-            return "Tuesday"
-        case 3:
-            return "Wednesday"
-        case 4:
-            return "Thursday"
-        case 5:
-            return "Friday"
-        case 6:
-            return "Saturday"
-        default:
+const getCurrentMeal = () => {
+    let meal = 0;
+    hour = hour + minute / 60;
+    for (let i = 0; i < 4; i++) {
+        if (hour < mealInfo[i].time) {
+            meal = i;
             break;
+        }
     }
+    hour > 21 ? day = (day + 1) % 7 : day;
```

Comment:
If dinner is over, it shows the next day's breakfast menu so if you don't want that behaviour, you can remove this line!